src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,47 @@
+import * as S from "./Navbar.Styles"
+import logo from "../../assets/logo.png"
+import profile from "../../assets/profile.png"
+import Button from "../Button/Button"
+import { useNavigate } from "react-router-dom"
+import SearchIcon from "@mui/icons-material/Search"
+
+const navMenu = ["게시물 생성", "DM", "알림", "로그인"]
```

Comment:
[ P3 ] 어쩌면 이 부분도 하나의 상수라고 할 수 있을것 같네요!
```suggestion
const NAV_MENU_LIST = ["게시물 생성", "DM", "알림", "로그인"]
```

Reply:
> [ P3 ] 어쩌면 이 부분도 하나의 상수라고 할 수 있을것 같네요!

맞아요!! 바로 반영해야겠습니다 ㅎㅎ